dwitter/static/main.css

```diff
@@ -461,6 +461,30 @@ iframe {
   }
 }
 
+
+.top-sort-list li .collapsible li {
+  display: block;
+  background: white;
+  border: 1px solid #ddd;
+  padding: 2px;
+}
+
+.top-sort-list li .collapsible {
+  position: absolute;
+  z-index: 10;
+  transition: opacity .1s ease-in-out, margin .1s step-end;
+  opacity: 0;
+  height: 0;
+}
+
+.top-sort-list li:hover>.collapsible,
+.top-sort-list li .collapsible:hover {
+  position: absolute;
+  display: block;
+  opacity: 1;
+  transition: opacity .1s ease-in-out, margin .1s step-start;
```

Comment:
I think you need to add a `margin-top: 0` here, and a `margin-top: -9999px;` in the block above.
Currently, the dropdown shows up when the mouse cursor passes below the top navigation bar.

Reply:
Thanks! You're right, that would've been really annoying. Nice catch.

Your suggestion fixes it, see new commit.